pkg/controller/defaults/placements.go

```diff
@@ -9,24 +9,25 @@ import (
 var (
 	// appLabelSelectorKey is common value for 'Key' field in 'LabelSelectorRequirement'
 	appLabelSelectorKey = "app"
-	// DaemonPlacements map contains the default placement configs for the
-	// various OCS daemons
-	DaemonPlacements = map[string]rook.Placement{
-		"all": rook.Placement{
-			NodeAffinity: &corev1.NodeAffinity{
-				RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
-					NodeSelectorTerms: []corev1.NodeSelectorTerm{
-						corev1.NodeSelectorTerm{
-							MatchExpressions: []corev1.NodeSelectorRequirement{
-								corev1.NodeSelectorRequirement{
-									Key:      NodeAffinityKey,
-									Operator: corev1.NodeSelectorOpExists,
-								},
-							},
+	// DefaultNodeSelector is the NodeAffinity to be used when labelSelector is nil
+	DefaultNodeSelector = &corev1.NodeAffinity{
```

Comment:
If it is `DefaultNodeAffinity`, why name it `DefaultNodeSelector`?

Reply:
Ah! Will fix that